docs/website-v2/docs/basics/prerequisites.md

```diff
@@ -39,7 +39,7 @@ allocation for the Docker engine to at least 4GB (Preferences -> Resources ->
 Advanced). If you are running into build/test failures with Go that report
 "signal killed", you likely need to increase Docker's allocated resources.
 
-![Increasing docker engine resources](assets/docker-config.png)
+![Increasing docker engine resources](/assets/docker-config.png)
```

Comment:
We don't support a version-1.0.0/assets anymore, however you can use relative paths to any assets, but those paths are resolved against the md file location, not the. docs/asset folder.

However if you put images in /static/assets/myImage.png you can use an absolute path like `/assets/myImage.png`

I thought it was simpler to add a leading / everywhere and move images to /static, but you can also rewrite all the image paths 😅 

Let me know if it is a problem, but it looks to me your images are mostly the same across versions and versioning them might be overkill?

Note: for me, all the images in the repo are broken, not sure where those images even came from as I don't find them in the Docusaurus v1 site